.github/workflows/Tests.yaml

```diff
@@ -642,45 +648,14 @@ jobs:
           git push origin HEAD:develop
           git push origin HEAD:release
           git push origin v$RELEASE_VERSION
-      - name: Tag and release
+      - name: Create release on GitHub
         uses: actions/create-release@v1
         env:
-          # If we need this event to trigger other workflows we could use a
-          # personal access token
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GH_TOKEN_RELEASE }}
         with:
           tag_name: v${{ env.RELEASE_VERSION }}
           release_name: Release ${{ env.RELEASE_VERSION }}
           body_path: /work/release_notes.md
-      # The release is now public on GitHub. We build the release documentation
-      # from the archive and also build an executable to make sure the archive
-      # is functional.
```

Comment:
Why are you removing this test?  It seems like a good sanity check.

Reply:
Two reasons (you tell me if you agree or disagree):

1. The unit tests now check that the archive extracted by formaline is functional. I think that's a similar enough test that this one is not so valuable anymore.
2. The code in this job will only run ~once a month, so it has a high likelihood to break e.g. when cmake configuration options change, when the name of the executable changes or the name of the unit test, none of which mean that the release is broken. I want to avoid that I have to deal with a failed release job every other month because I have to update this cmake, make and run code.

My plan is to add a separate `on: release` workflow at some point that builds and deploys a Docker container with compiled executables from the release, similar to what https://github.com/nilsleiffischer/spectre-deploy does.

Reply:
Yes, you're right, they are similar enough.